extensions/amp-story/1.0/test/test-amp-story.js

```diff
@@ -1554,17 +1589,6 @@ describes.realWin(
       });
 
       describe('touch events handlers', () => {
-        const getTouchOptions = (x, y) => {
-          const touch = new Touch({
-            target: story.element,
-            identifier: Date.now(),
-            clientX: x,
-            clientY: y,
-          });
-
-          return {touches: [touch], bubbles: true};
-        };
-
```

Comment:
Moved getTouchOptions to outside this scope to be able to use it on the other test.